native/cocos/ui/edit-box/EditBox-mac.mm

```diff
@@ -149,6 +150,9 @@ - (NSAttributedString *)attributedStringForObjectValue:(id)anObject withDefaultA
  Implementation of global helper functions.
  ************************************************************************/
 namespace {
+
+static cc::events::Resize::Listener resizeListener;
```

Comment:
Where to initialize it?

Reply:
Events internal implementation by @PatriceJiang